src/core/Akka.Remote/Endpoint.cs

```diff
@@ -1100,23 +1030,27 @@ private IActorRef StartReadEndpoint(AkkaProtocolHandle handle)
         {
             var newReader =
                 Context.ActorOf(RARP.For(Context.System)
-                .ConfigureDispatcher(
-                    EndpointReader.ReaderProps(LocalAddress, RemoteAddress, Transport, Settings, _codec, _msgDispatcher,
-                        Inbound, (int)handle.HandshakeInfo.Uid, _receiveBuffers, _reliableDeliverySupervisor)
-                        .WithDeploy(Deploy.Local)),
+                    .ConfigureDispatcher(
+                        EndpointReader.ReaderProps(LocalAddress, RemoteAddress, Transport, Settings, _codec, _msgDispatcher,
+                            Inbound, (int)handle.HandshakeInfo.Uid, _receiveBuffers, _reliableDeliverySupervisor)
+                            .WithDeploy(Deploy.Local)),
                     string.Format("endpointReader-{0}-{1}", AddressUrlEncoder.Encode(RemoteAddress), _readerId.Next()));
             Context.Watch(newReader);
             handle.ReadHandlerSource.SetResult(new ActorHandleEventListener(newReader));
             return newReader;
         }
 
+        /// <summary>
+        /// TODO: this varies from the JVM but I can't tell if it's significant
```

Comment:
@rogeralsing care to comment on some of the API differences in this method from the JVM? I don't know enough to tell if they're significant or not. 

https://github.com/akka/akka/blob/master/akka-remote/src/main/scala/akka/remote/Endpoint.scala#L868